tutorials/data_management/cloud_storage/binding_with_azure/chapter.md

```diff
@@ -54,18 +62,28 @@ def main(event: func.EventGridEvent):
         else:
             dataset.items.delete(filename=file_name)
 ```
-4. Deploy the code to the function app you created.  
+4. Deploy the code to the function app you created [azure docs](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-python?pivots=python-mode-configuration#deploy-the-project-to-azure)  
 5. In VS code go to view tab -> Command Palette -> Azure Functions: Upload Local Settings  
 6. Go to the Function App -> Select your function -> Configuration (Under Settings section)  
        * add the 4 secrets vars DATASET_ID, DTLPY_USERNAME, DTLPY_PASSWORD, CONTAINER_NAME (your container that want to trigger it)  
-7. Go to Function App -> select your function -> go to Functions tag and select your function -> Integration ->  
+            * for get the DTLPY_USERNAME, DTLPY_PASSWORD you need to create a bot to your project useing this code  
+
+```python
+import dtlpy as dl
+project = dl.projects.get(project_name='project name')
+bot = project.bots.create(name='bot name', return_credentials=True)
+print('username', bot.id)
+print('password', bot.password)
+```
+7. Go to Function App -> select your function -> navigate in the sidebar to the functions tab and select your function -> Integration ->  
```

Comment:
Go to Function App -> Select your function -> Navigate in the sidebar to the `**functions**` tab and select your **trigger** function -> Integration ->